deploy/sre-prometheus/100-node-unschedulable.PrometheusRule.yaml

```
@@ -14,7 +14,7 @@ spec:
       expr: (kube_node_spec_unschedulable > 0 and on(node) kube_node_role{role="master"}) or (kube_node_spec_unschedulable > 0 unless ignoring(node,container,endpoint,job,namespace,service) sum(mapi_machine_set_status_replicas{name=~".*upgrade$"}) > 0)
       for: 61m
       labels:
-        severity: critical
+        severity: Warning
         namespace: openshift-monitoring
```

```suggestion
        namespace: openshift-monitoring
        send_managed_notification: "true"
        managed_notification_template: "KubeNodeUnschedulableSRE"  
```

Per the other comment, we should be able to just add these labels here.

I wonder if we want to reduce the time until the alert fires here as well. 60 minutes was to protect SREs from flapping alerts - but a customer may want to know earlier since it directly affects their cloud spend. Maybe 30 minutes here? I don't feel strongly enough to make that a required change if there's any argument against it though.